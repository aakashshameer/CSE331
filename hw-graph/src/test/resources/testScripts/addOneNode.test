#Test to create a node in the graph

#Create a graph
CreateGraph dgraph

#Add a node
AddNode dgraph a

#Print nodes in the graph
ListNodes dgraph

#Print outgoing edges from a
ListChildren dgraph a