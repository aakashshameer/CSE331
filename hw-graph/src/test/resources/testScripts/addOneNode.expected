#Test to create a node in the graph

#Create a graph
created graph dgraph

#Add a node
added node a to dgraph

#Print nodes in the graph
dgraph contains: a

#Print outgoing edges from a
the children of a in dgraph are: